src/main/autoupdater/autoUpdater.ts

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import path from 'path';
+
+import {dialog, ipcMain, app, nativeImage} from 'electron';
+import log from 'electron-log';
+import {autoUpdater, UpdateInfo} from 'electron-updater';
+
+import {displayUpgrade, displayRestartToUpgrade} from 'main/notifications';
+
+import * as WindowManager from '../windows/windowManager';
+
+import {CANCEL_UPGRADE, UPDATE_AVAILABLE} from 'common/communication';
+
+//const NEXT_NOTIFY = 86400000; // 24 hours
+//const NEXT_CHECK = 3600000;
+const NEXT_CHECK = 60000; // todo: remove me
+const NEXT_NOTIFY = 60000;
```

Comment:
same as above, the right values are the ones commented, once we are done with testing this should be removed and the two lines above uncommented.